CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Guide to Making Developer Contributions
+
+## Overview:
+
+A CSV with package name and version is uploaded by the user, 
+then the risk of the packages are assessed using `riskmetric`.
+[TODO this is shakey for me.... where is this function actually called... 
```

Comment:
The `riskmetric` package is loaded in the `setup.R` file (lines 43-47). Then, in the `./Modules/dbupload.R` file, the function  `metric_mm_tm_Info_upload_to_DB` calls `riskmetric` and saves the different metric results on the db.